_posts/2016-08-04-buying-better-digital-products-part-3-mapping-user-stories.md

```diff
@@ -62,7 +62,7 @@ The diagram below lists out the tasks, but also includes a breakdown of each tas
 
 Release planning is traditionally used to determine what is feasible with what is expected and establish a baseline plan to get there and commit to deliver. Release cycles are important because they allow for the building of all the major and necessary features a little at a time (as opposed to a feature at a time). This approach allows for usability testing and hypothesis validation early. 
 
-The MVP, as mapped in [day two of the inception workshop](https://18f.gsa.gov/2016/07/26/buying-better-digital-products-part-2-setting-the-product-vision-and-strategy/), should match the user story map. We  defined the MVP in more detail by using tape to mark a horizontal line on the user stories. We then moving stickies around so that the tasks and activities above the line were to be included in the MVP. As the build of the MVP actually goes underway, it’s important for teams to continue to negotiate, re-plan, and undergo continuous review to make adjustments after each set of sprints.
+The MVP, as mapped in [day two of the inception workshop](https://18f.gsa.gov/2016/07/26/buying-better-digital-products-part-2-setting-the-product-vision-and-strategy/), should match the user story map. We  defined the MVP in more detail by using tape to mark a horizontal line on the user stories. We then moved stickies around so that the tasks and activities above the line were to be included in the MVP. As the build of the MVP actually goes underway, it’s important for teams to continue to negotiate, re-plan, and undergo continuous review to make adjustments after each set of sprints.
```

Comment:
Also hard to read the diff. I basically changed the beginning of the third sentence: "We then ~~moving~~ moved stickies around".

That said, the sentence is still confusing to me, but I'm not sure how to make it easier to understand.